src/vgl/texture.js

```diff
@@ -138,6 +153,36 @@ vgl.texture = function () {
     return false;
   };
 
+  /**
+   * Set Raw Texture data using Uint8Array
+   *
+   * @param {object} texture texture object to load.
+   *   type: 'RGB' | 'RGBA' | 'Luminance' | 'LuminanceAlpha'.
```

Comment:
If you pass RGB, doesn't this still expect data in RGBA?  Should we remove the RGB option if it isn't directly supported?

Reply:
And, I _think_ the jsdoc of these sub properties should be 
```
   * @param {object} texture texture object to load.
   * @param {string?} texture.type One of 'RGB' | 'RGBA' | 'Luminance' | 'LuminanceAlpha'.
   * @param {Uint8Array} texture.data Array representing the format based on the type
   * @param {number} texture.width width of the texture
   * @param {number} texture.height height of the texture
```

Reply:
Yeah, I'll remove RGB, it was part of my early testing. Having RGBA should cover most of the use cases.